Bottom: d0a0b66dff02dfa4876167fefdf76972499cc804
Top:    d687c2a7b298467031ac36a0a51054692cfcabad
Author: Jared Casper <jaredcasper@gmail.com>
Date:   2011-01-07 16:39:49 -0800

Heed and remove warning issued by autoconf 2.68.

Starting in autoconf 2.68, "the macros AC_PREPROC_IFELSE,
AC_COMPILE_IFELSE, AC_LINK_IFELSE, and AC_RUN_IFELSE now warn if the
first argument failed to use AC_LANG_SOURCE or AC_LANG_PROGRAM to
generate the conftest file contents."

Closes-bug: lp-700180

Reviewed-by: Peter Clifton <pcjc2@cam.ac>


---

diff --git a/configure.ac b/configure.ac
index 13e0e07..ca2dbe4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -264,7 +264,7 @@ if test "x$GCC" = "xyes"; then
 AC_MSG_CHECKING([If the compiler accepts -rdynamic])
 old_LDFLAGS="$LDFLAGS"
 LDFLAGS="$LDFLAGS -rdynamic"
-AC_LINK_IFELSE([int main(){}],
+AC_LINK_IFELSE([AC_LANG_PROGRAM()],
 	[AC_MSG_RESULT([yes])],
 	[LDFLAGS="$old_LDFLAGS"
 	AC_MSG_RESULT([no])
